cli/test/README.md

```diff
@@ -27,21 +27,65 @@ This document includes CLI commands and the expected CLI output in order to help
+|`fastddsqosprof file.xml set datareader[profile].external_locators`|**PENDING** (currently `DataReader default attribute configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set datareader[profile].history_memory_policy`|**PENDING** (currently `DataReader history memory policy configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set datareader[profile].ignore_non_matching_locators`|**PENDING** (currently `DataReader ignore non matching locators configuration flag not yet supported`)|N/A|
+|`fastddsqosprof file.xml set datareader[profile].locators`|**PENDING** (currently `DataReader locators configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set datareader[profile].property`|**PENDING** (currently `DataReader properties configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set datareader[profile].qos`|**PENDING** (currently `DataReader QoS configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set datareader[profile].times`|**PENDING** (currently `DataReader time related parameters configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set datareader[profile].user_defined_id`|**PENDING** (currently `DataReader user defined ID configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set datareader[profile].other`|`ERROR: other element not recognized`|`DATAREADER_USAGE`|
+|`fastddsqosprof file.xml set datareader[profile].other --help`|N/A|`DATAREADER_USAGE`|
+|`fastddsqosprof file.xml set datawriter -h`|N/A|`DATAWRITER_USAGE`|
+|`fastddsqosprof file.xml set datawriter --help`|N/A|`DATAWRITER_USAGE`|
+|`fastddsqosprof file.xml set datawriter help`|N/A|`DATAWRITER_USAGE`|
+|`fastddsqosprof file.xml set datawriter argument1 argument2 help`|N/A|`DATAWRITER_USAGE`|
+|`fastddsqosprof file.xml set datawriter.default_profile`|`ERROR: datawriter requires a profile name` [^1]|`DATAWRITER_USAGE`|
+|`fastddsqosprof file.xml set datawriter[profile]`|`ERROR: datawriter must not be FINAL element` [^1]|`DATAWRITER_USAGE`|
+|`fastddsqosprof file.xml set datawriter["a profile"]`|`ERROR: datawriter must not be FINAL element` [^1]|`DATAWRITER_USAGE`|
+|`fastddsqosprof file.xml set datawriter[profile].allocations`|**PENDING** (currently `DataWriter allocations configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set datawriter[profile].binary_property`|**PENDING** (currently `DataWriter binary properties configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set datawriter[profile].default_profile`|**PENDING** (currently `DataWriter default attribute configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set datawriter[profile].entity_id`|**PENDING** (currently `DataWriter entity ID configuration not yet supported`)|N/A|
+|`fastddsqosprof file.xml set datawriter[profile].expect_inline_qos`|`ERROR: expect_inline_qos element not recognized`|`DATAWRITER_USAGE`|
```

Comment:
Perhaps this should be treated as `other` as in the case below. I would not include it here
```suggestion
```

Reply:
I think it is important to check this specific value because it is valid for DataReaders but not for DataWriters.

Reply:
Allright, lets keep that to reinforce the element difference!